### ISSUE TITLE
riot-desktop sync custom dictionary spellchecker words over Matrix

### ISSUE THREAD

**Issue by t3chguy** [vector-im/element-meta#464]

**Comment by daniellekirkwood** [comment]
@t3chguy - is this still relevant? Is there more info you could provide? Should it be a discussion, does it need labels?

**Comment by t3chguy** [comment]
Yes it is relevant, you can save words to your custom dictionary, we should sync that between your clients.

<img width="252" alt="image" src="https://user-images.githubusercontent.com/2403652/183863650-e86d356f-00f1-4c6d-b815-070e82bb8da1.png">

**Comment by t3chguy** [comment]
I'm not sure it can feasibly be done cross platform, in web we don't control spellchecking and to my knowledge in android it is system wide, not app specific

**Comment by SimonBrandner** [comment]
It should be possible on desktop fwiw

**Comment by t3chguy** [comment]
Yes, that's where this issue originated, hence the title of `riot-desktop`

**Comment by SimonBrandner** [comment]
> Yes, that's where this issue originated, hence the title of `riot-desktop`

Missed that somehow :sweat_smile:
